packages/ember-routing/lib/system/route.js

```diff
@@ -1770,16 +1770,20 @@ let Route = EmberObject.extend(ActionHandler, Evented, {
     Example
 
     ```javascript
-    App.Router.map(function() {
-        this.route('post', { path: '/post/:post_id' }, function() {
-          this.route('comments', { resetNamespace: true });
-        });
+    // app/router.js
+    Router.map(function() {
+      this.route('post', { path: '/post/:post_id' }, function() {
+        this.route('comments', { resetNamespace: true });
+      });
     });
 
-    App.CommentsRoute = Ember.Route.extend({
-        afterModel: function() {
-          this.set('post', this.modelFor('post'));
-        }
+
```

Comment:
I'd maybe have two different snippets to make it more evident they're different files.